### PR TITLE
Support telemetry

### PR DESCRIPTION
**Pull request by FelonEkonom**

### CODE REVIEW

File: lib/videoroom/application.ex [hunk 1]
```diff
@@ -12,6 +15,10 @@ defmodule VideoRoom.Application do
     create_integrated_turn_cert_file()
 
     children = [
+      %{
+        id: Reporter,
+        start: {Reporter, :start_link, [[metrics: Engine.Metrics.metrics()], [name: Reporter]]}
+      },
```
Comment (mickel8) [inline lib/videoroom/application.ex:21]: Will it work?


```suggestion
      {Reporter, :start_link, [[metrics: Engine.Metrics.metrics()], [name: Reporter]]}

File: priv/static/grafana_dashboard.json [hunk 2]
```diff
@@ -0,0 +1,459 @@
+{
```
Comment (mickel8) [inline priv/static/grafana_dashboard.json:1]: Are we using this?

Comment (mickel8) [inline priv/static/grafana_dashboard.json:1]: Are we using this?

File: priv/static/promehteus.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (mickel8) [inline priv/static/promehteus.yml:1]: As above. Is it used?

Comment (mickel8) [inline priv/static/promehteus.yml:1]: As above. Is it used?

**Review by mickel8 — COMMENTED** [review]
I would add an endpoint that will return stats for current room as a json. Just for now and for debugging purposes. We can get rid of it once we implement storing stats in db.

File: lib/videoroom_web/router.ex [hunk 4]
```diff
@@ -14,6 +14,8 @@ defmodule VideoRoomWeb.Router do
 
     get("/room/:room_id", RoomController, :index)
 
+    get("/room/:room_id/scrape", RoomController, :scrape)
```
Comment (mickel8) [inline lib/videoroom_web/router.ex:17]:
```suggestion
    get("/room/:room_id/stats", RoomController, :scrape)
```

**Review by mickel8 — APPROVED** [review]

File: lib/videoroom/metrics.ex [hunk 5]
```diff
@@ -0,0 +1,20 @@
+defmodule VideoRoom.Metrics do
```
Comment (mat-hek) [inline lib/videoroom/metrics.ex:1]: This module doesn't feel really beneficial, I'd remove it

**Review by mat-hek — APPROVED** [review]

**Pull request merged**